cluster/ubuntu/util.sh

```diff
@@ -358,13 +358,13 @@ function provision-master() {
                             create-kube-apiserver-opts "${SERVICE_CLUSTER_IP_RANGE}" "${ADMISSION_CONTROL}" "${SERVICE_NODE_PORT_RANGE}"; \
                             create-kube-controller-manager-opts "${NODE_IPS}"; \
                             create-kube-scheduler-opts; \
-                            create-flanneld-opts "127.0.0.1"; \
                             sudo -p '[sudo] password to start master: ' cp ~/kube/default/* /etc/default/ && sudo cp ~/kube/init_conf/* /etc/init/ && sudo cp ~/kube/init_scripts/* /etc/init.d/ ;\
                             sudo groupadd -f -r kube-cert; \
                             ${PROXY_SETTING} sudo -E ~/kube/make-ca-cert.sh ${MASTER_IP} IP:${MASTER_IP},IP:${SERVICE_CLUSTER_IP_RANGE%.*}.1,DNS:kubernetes,DNS:kubernetes.default,DNS:kubernetes.default.svc,DNS:kubernetes.default.svc.cluster.local; \
                             sudo mkdir -p /opt/bin/ && sudo cp ~/kube/master/* /opt/bin/; \
                             sudo service etcd start; \
-                            sudo FLANNEL_NET=${FLANNEL_NET} -b ~/kube/reconfDocker.sh "a";"
+                            sudo FLANNEL_NET=${FLANNEL_NET} ~/kube/reconfDocker.sh "a"; \
+                            create-flanneld-opts "127.0.0.1";"
```

Comment:
I have just realized that you were talking about the part of the code related to the provision-master function.
The order of that part of code was fine and I just reordered it to keep the symmetry regarding the provision-masterandnode function.
In addition we noticed that executing reconfDocker.sh in background was preventing the script from correctly create/update /etc/default/docker.
Is your env a ("ai" "i" ... "i") cluster?